recipes/freeimage/all/conandata.yml

```
@@ -18,3 +18,4 @@ patches:
     - patch_file: "patches/012_use-functions-to-override-libtiff-warning-error-handlers.patch"
     - patch_file: "patches/013_use-typedef-as-already-declared.patch"
     - patch_file: "patches/014_no_auto_ptr.patch"
+    - patch_file: "patches/015_fix_msvc_cpp_20.patch"
```

```suggestion
    - patch_file: "patches/015_fix_msvc_cpp_20.patch"
      patch_type: "official"
      patch_source: "https://github.com/WinMerge/freeimage/pull/20"
      patch_description: "Fix compilation with MSVC and C++20"
```

Please, always add information about the patch, so we track it.